synbiohub_adapter/upload_sbol/upload_sbol.py

```diff
@@ -62,20 +64,37 @@ def main(args=None):
 
 
 class SynBioHub():
-    def __init__(self, url, email, password, sparql):
+    def __init__(self, url, email, password, sparql, spoofed_url=None):
         url = url.rstrip('/')
         self.url = url
+        self.email = email
         self.part_shop = PartShop(url)
         self.part_shop.login(email, password)
         response = requests.post(url + '/login',
                                  headers={'Accept': 'text/plain'},
                                  data={'email': email, 'password': password})
         self.token = response.content.decode('UTF-8')
         self.sparql = sparql
+        self.spoofed_url = spoofed_url
+
+    def __resubmit_collection(self, doc, collection_id, overwrite):
+        if overwrite:
+            if self.spoofed_url:
+                collection_uri = '/'.join([self.spoofed_url, 'user', self.email, collection_id,
+                                          collection_id + '_collection', '1'])
+            else:
+                collection_uri = '/'.join([self.url, 'user', self.email, collection_id,
+                                          collection_id + '_collection', '1'])
+
+            response = self.part_shop.submit(doc, collection_uri, 1)
+
+            print(response)
+        else:
+            raise DuplicateCollectionError(doc.displayId, doc.version)
```

Comment:
This new method appears to be unused. Can it be deleted or did I miss a reference to it?

Reply:
I removed this method.